lib/timeutils/format.c

```diff
@@ -97,7 +97,7 @@ append_format_wall_clock_time(const WallClockTime *wct, GString *target, gint ts
   switch (ts_format)
     {
     case TS_FMT_BSD:
-      g_string_append(target, month_names_abbrev[wct->wct_mon]);
+      g_string_append_len(target, month_names_abbrev[wct->wct_mon], 3);
```

Comment:
Here and below you could also use the newly introduced constant names.

Reply:
I remember doing those, that's why I introduced the constants... Ah, I did that in macros.c, good catch.
fixed.